packages/app/src/embed/components/Actions/index.js

```diff
@@ -101,7 +79,7 @@ export function GlobalActions({
               : `${sandboxUrl(sandbox)}?from-embed`
           }
         >
-          {openWordingB ? 'Open and Fork' : 'Open Sandbox'}
+          Open Sandbox
```

Comment:
@tromika is this the winner right?

Reply:
@danilowoz yes, this is the original version. Unfortunately, we couldn't make an impact here.